backend/app/services/prediction_service.py

```
@@ -858,6 +870,54 @@ def predict(
 
         return result
 
+    def _predict_ensemble(
```

Pylint (3.3.7)

[refactor] Too many arguments (8/5)

(R0913)